docker/scripts/start-spark.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+. "/opt/spark/bin/load-spark-env.sh"
+# When the spark work_load is master run class org.apache.spark.deploy.master.Master
+if [ "$SPARK_WORKLOAD" == "master" ];
+then
+
+export SPARK_MASTER_HOST=`hostname`
+
+cd /opt/spark/bin && ./spark-class org.apache.spark.deploy.master.Master --ip $SPARK_MASTER_HOST --port $SPARK_MASTER_PORT --webui-port $SPARK_MASTER_WEBUI_PORT >> $SPARK_MASTER_LOG
```

Comment:
Spark's quickstart [1] simply uses "./sbin/start-master.sh" and "./sbin/start-worker.sh" to start master and worker. It seems simpler to follower that approach.

How about just using something like `./sbin/start-master.sh && ./sbin/start-worker.sh` in the docker compose file instead of adding this script?

[1] https://spark.apache.org/docs/latest/spark-standalone.html#starting-a-cluster-manually